micro_ros_setup/config/freertos/crazyflie21/client-colcon.meta

```diff
@@ -9,21 +9,12 @@
         },
         "microxrcedds_client":{
             "cmake-args":[
-                "-DUCLIENT_PIC=OFF",
-                "-DEXTERNAL_TRANSPORT_HEADER=../../../crazyflie_microros_extensions/src/crazyflie_transport.h",
-                "-DEXTERNAL_TRANSPORT_SRC=../../../crazyflie_microros_extensions/src/crazyflie_transport.c"
+                "-DUCLIENT_PIC=OFF"
             ]
         },
         "rmw_microxrcedds":{
             "cmake-args":[
-                "-DRMW_UXRCE_TRANSPORT=custom",
-                "-DRMW_UXRCE_MAX_NODES=1",
-                "-DRMW_UXRCE_MAX_PUBLISHERS_X_NODE=2",
-                "-DRMW_UXRCE_MAX_SUBSCRIPTIONS_X_NODE=1",
-                "-DRMW_UXRCE_MAX_SERVICES_X_NODE=1",
-                "-DRMW_UXRCE_MAX_CLIENTS_X_NODE=1",
-                "-DRMW_UXRCE_MAX_HISTORY=1",
-                "-DRMW_UXRCE_XML_BUFFER_LENGTH=400"
+                "-DRMW_UXRCE_TRANSPORT=custom"
```

Comment:
It must be now `custom_serial`.

Reply:
Please check https://github.com/micro-ROS/micro-ros-build/pull/84